### DISCUSSION TITLE
window.chrome.webview.addEventListener Uncaught TypeError

### DISCUSSION THREAD

**Discussion by jinnyum** [MicrosoftEdge/WebView2Feedback#4035] [Q&A]
window.chrome.webview.addEventListener outputs an error when an ad appears on YouTube.
If ads are not displayed when viewing YouTube, no error is displayed
is this a bug?

This happens in the official webview sample code.
https://github.com/MicrosoftEdge/WebView2Samples/tree/main/GettingStartedGuides/Win32_GettingStarted

framework : win32
sdk : 10.0.22000.0
webview2 runtime : 110.0.1587.69
os : windows 10 Pro 21H2

![error1](https://user-images.githubusercontent.com/38673939/230120686-e21cbd7d-38d6-4d17-b63f-6105d68196ff.png)
![error2](https://user-images.githubusercontent.com/38673939/230120697-4180540c-5878-4832-b3e1-dbcbd929d9e7.png)

**Comment by novac42** [comment]
Thanks for reporting the issue. I've assigned this to a dev that can help follow up on this.
